build.cake

```diff
@@ -20,6 +21,13 @@ Task("Default")
     .IsDependentOn("Ckan")
     .IsDependentOn("Netkan");
 
+
+Task("Debug")
+    .IsDependentOn("Default");
+
+Task("Release")
+    .IsDependentOn("Default");
+
 Task("Ckan")
```

Comment:
Is it possible to make these task names case-agnostic?

Reply:
They already are.